setup.py

```diff
@@ -281,6 +332,7 @@ def check_for_openmp(cc_var):
     #},
 
     ext_modules = extensions,
-    #cmdclass={'build_ext':build_ext},
+    cmdclass={'build_ext':build_ext,
+              'clean':CleanCommand},
```

Comment:
This adds the possibility to do
`python setup.py clean`
and that it actually cleans something...